cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/css/workflow.css

```diff
@@ -0,0 +1,36 @@
+.steps-box p {margin:0; padding-bottom:1em}
```

Comment:
This file is really difficult to read. Could you format it in a better way?

```
.steps-box p {
    margin:0; 
    padding-bottom:1em
}
```
That is much better than the one-liner style.